spec/functional/templating_spec.rb

```diff
@@ -30,4 +30,88 @@
       EMR
     )).to eq('<h1>Hello, Dave</h1>')
   end
+
+  it 'works in multiline literals' do
+    expect(convert(
```

Comment:
Can you put the `convert(` on a new line? The pre-commit rubocop was complaining about this for me.